depends/Makefile

```diff
@@ -205,6 +209,7 @@ $(host_prefix)/share/config.site : config.site.in $(host_prefix)/.stamp_$(final_
             -e 's|@LDFLAGS@|$(strip $(host_LDFLAGS) $(host_$(release_type)_LDFLAGS))|' \
             -e 's|@no_qt@|$(NO_QT)|' \
             -e 's|@no_zmq@|$(NO_ZMQ)|' \
+            -e 's|@disable_bip70@|$(PROTOBUF)|' \
```

Comment:
16871: this makes no sense now - you are _disabling_ bip70 when $PROTOBUF is set to 1, should be the other way around. We start to diverge from the original PR no matter how we try to fix it, so I'd propose to drop 16871 for now and either
a) backport bip70 related PRs in the right order ("disable -> enable" (bip70 disabled by default) first and then this one) or
b) backport this one again but with `$NO_PROTOBUF`/`@no_protobuf@` patch on top to make it similar to other modules

Reply:
I tried going for option b, hopefully this version makes sense and has no error